### ISSUE TITLE
Scrollbar is not hide from accessibility perspective when autoHide is true.

### ISSUE THREAD

**Issue by CanadaBear** [malte-wessel/react-custom-scrollbars#420]
when autoHide is set as true, both vertical and horizonal scrollbars should be set as aria-hidden as true as well, otherwise it will still be accessed by accessibility tool.
